include/links-permalinks.php

```diff
@@ -164,4 +201,17 @@ public function get_rewrite_rules_filters() {
 		 */
 		return apply_filters( 'pll_rewrite_rules', $types );
 	}
+
+	/**
+	 * Removes hooks to filter rewrite rules, called when switching blog @see {PLL_Base::switch_blog()}.
+	 *
+	 * @since 3.5
+	 *
+	 * @return void
+	 */
+	public function remove_filters() {
+		parent::remove_filters();
```

Comment:
Not needed as the parent method does nothing. 
Note for my future self. It's useful anyway if someday the parent method does something as we may not think to look at this child method at this future time.

Reply:
That's what I had in mind.